.github/workflows/release.yml

```diff
@@ -28,48 +26,50 @@ jobs:
       uses: joschi/setup-jdk@v2.3.0
       with:
         java-version: ${{ matrix.java }}
-    - name: Build and Release for Git Tag
+    - name: Build and Release
       env:
         BASE: origin/${{ github['base_ref'] }}
         GITHUB_TOKEN: ${{ secrets.DOCS_SITE_TOKEN }}
         GITHUB_REPOSITORY: ${{ secrets.GITHUB_REPOSITORY }}
         GITHUB_ACTOR: ${{ secrets.GITHUB_ACTOR }}
-        BINTRAY_USERNAME: ${{ secrets.BINTRAY_USERNAME }}
-        BINTRAY_PASSWORD: ${{ secrets.BINTRAY_PASSWORD }}
-        CODECOV_RUNME: ${{ secrets.CODECOV_RUNME }}
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
         DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
-        CI: true
+        GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+        GPG_KEYNAME: ${{ secrets.GPG_KEYNAME }}
+        GPG_KEY_FILE: ${{ secrets.GPG_KEY_FILE }}
+        OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+        OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
```

Comment:
what is OSSRH?

Reply:
sonatype ... Sonatype OSSRH (OSS Repository Hosting)

Reply:
ah yes, thats ringing a bell now